preload.js

```diff
@@ -201,6 +201,7 @@ window.Signal.Startup = require('./js/modules/startup');
 
 window.Signal.Types = {};
 window.Signal.Types.Attachment = Attachment;
+window.Signal.Types.AttachmentTS = require('./ts/types/Attachment');
```

Comment:
Bleh. Maybe what we need is a function that will merge the two objects together?

Reply:
In my changes view I'm still seeing the addition of `Signal.Types.AttachmentTS` in `preload.js`...